src/main/api/2/studio-api-2.yaml

```diff
@@ -1100,6 +1100,89 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /users/me/properties:
+    get:
+      tags:
+        - users
+      summary: Retrieve properties for the current user
+      description: 'Required Permission: "LOGGED_IN"'
+      operationId: getUserProperties
+      parameters:
+        - name: siteId
+          schema:
+            type: string
+          in: query
+          description: The id of the site (defaults to global)
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  properties:
+                    type: object
+                    additionalProperties:
+                      type: string
+                    description: The properties of the current user
```

Comment:
1. Isn't this an array of objects? (e.g. array of UserProperty schemas?)

2. What is the property object like? Based of the db schema, it'd be
```ts
interface UserProperty {
  id: number;
  key: string;
  value: string;
  siteId: string;
}
```

Reply:
It should be similar to what @rart shows, and similar to the DB.

Reply:
I don't think this API should expose the full db table, I was thinking a very simple key-property GET/UPDATE/DELETE

Reply:
i don’t mind not getting the db id but feels like at the min we need the key, value, site triad.

Reply:
response: {
  site1: {
    key: value
    key: value
  }
  site2: {
    key: value
    key: value
  }
}